t/19-isotovideo-command-processing.t

```diff
@@ -394,6 +394,34 @@ subtest signalhandler => sub {
     } qr/isotovideo received signal INT/, 'Signal logged';
     is($last_signal, 'INT', 'Event emitted');
 };
+subtest 'Check exit_code_from_test_results' => sub {
+    my $runner = OpenQA::Isotovideo::Runner->new;
+    use Mojo::File 'path';
+    use Mojo::JSON 'encode_json';
+    my %fake_results = ();
+
+    subtest 'no test scheduled' => sub {
+        my $path_mock = Test::MockModule->new('Mojo::File');
+        $path_mock->redefine('path' => sub { 0 });
+        is($runner->exit_code_from_test_results(), 100, 'EXIT_STATUS_ERR_NO_TESTS returns if no results');
+    };
+
+    subtest 'tests scheduled' => sub {
+        %fake_results = ("result" => "failing");
+        my $json = encode_json(\%fake_results);
```

Comment:
Unless I'm overlooking something here you can probably remove the `%fake_results;` variable completely:
```suggestion
        my $json = encode_json({result => 'failing'});
```

Reply:
Done